test/controllers/api/v2/errata_controller_test.rb

```diff
@@ -40,35 +40,33 @@ def setup
     def test_index
       get :index, :repository_id => @repo.id
       assert_response :success
-      assert_template %w(katello/api/v2/errata/index)
+      assert_template "katello/api/v2/errata/index"
 
       get :index
       assert_response :success
-      assert_template %w(katello/api/v2/errata/index)
+      assert_template "katello/api/v2/errata/index"
 
       get :index, :organization_id => @repo.organization.id
       assert_response :success
-      assert_template %w(katello/api/v2/errata/index)
+      assert_template "katello/api/v2/errata/index"
     end
 
     def test_index_with_content_view_version
       @content_view_version = ContentViewVersion.first
-      ContentViewVersion.expects(:readable).returns(stub(:find_by_id => @content_view_version))
```

Comment:
This may have been me as well, it was an "unexpected invocation" I believe, and wasn't necessary for the test. I am not sure why content view version would have to be stubbed to begin with because it is all in AR afaik